src/components/Button/Button.js

```diff
@@ -13,8 +23,23 @@ class Button extends Component {
   };
 
   render() {
+    let classes = classNames(
+      'sd-button',
+      'ui',
+      'button',
+      this.props.color,
+      this.props.size,
+      {animated: this.props.animated},
+      {basic: this.props.basic},
+      {children: this.props.children},
+      {disabled: this.props.disabled},
+      {emphasis: this.props.emphasis},
+      {fluid: this.props.fluid},
+      {inverted: this.props.inverted},
+      {loading: this.props.loading}
+    );
     return (
-      <button className='sd-button ui blue button' {...this.props}>
+      <button className={classes} {...this.props}>
```

Comment:
Don't spread props since there are many props that are not applicable here, like color and size.